src/Illuminate/Support/helpers.php

```diff
@@ -172,6 +172,20 @@ function array_has($array, $keys)
     }
 }
 
+if (! function_exists('array_keys_exits')) {
+    /**
+     * Determines if keys of the provided array exist in another array.
+     *
+     * @param  array  $needed
+     * @param  array  $provided
+     * @return bool
+     */
+    function array_keys_exits($needed, $provided)
```

Comment:
Same

Reply:
Maybe we can keep the conversation going at a single place (in the first thread above) ?

Reply:
There appears to be a typo in the function name here (`exits` vs `exists`). 

In any case, you can't redeclare array_key_exists. It is a php built-in function. Even if you could, it would cause severe problems as people expecting to call php's function would end up calling this instead.

Reply:
@36864 oops 🙈 thanks @36864 :)